lib/protobuf/rpc/connectors/base.rb

```diff
@@ -104,6 +104,7 @@ def parse_response
           # Parse out the raw response
           @stats.response_size = @response_data.size unless @response_data.nil?
           response_wrapper = ::Protobuf::Socketrpc::Response.decode(@response_data)
+          @stats.server = response_wrapper.server if response_wrapper.field?(:server)
```

Comment:
This will always be true since they need to have the latest version with the new response proto to exec this code, right?